MobileCenter/MobileCenter/Internals/Utils/MSUtility+File.m

```diff
@@ -0,0 +1,78 @@
+#import "MSLogger.h"
+#import "MSMobileCenterInternal.h"
+#import "MSUtility+File.h"
+
+/*
+ * Workaround for exporting symbols from category object files.
+ */
+NSString *MSUtilityFileCategory;
+
+@implementation MSUtility (File)
+
++ (BOOL)createFileAtURL:(NSURL *)fileURL {
+  if (fileURL) {
+
+    // Check if file already exists.
+    if ([fileURL checkResourceIsReachableAndReturnError:nil]) {
+      return YES;
```

Comment:
I misunderstood this method, I thought it is writing something to the file below but it just writes an empty file. Yeah.. Let's don't log it.